File: Facebook.NET - VB/Contracts/GraphAPI.Achievement.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Achievement
            <DataMember(Name:="id")>
            Public Property ID() As String
            <DataMember(Name:="url")>
            Public Property Url() As String
            <DataMember(Name:="type")>
            Public Property Type() As String
            <DataMember(Name:="title")>
            Public Property Title() As String
        End Class
    End Class
End Namespace

File: Facebook.NET/Contracts/GraphAPI.Achievement.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Achievement
        {
            [DataMember(Name = "id")]
            public string ID { get; set; }

            [DataMember(Name = "url")]
            public string Url { get; set; }

            [DataMember(Name = "type")]
            public string Type { get; set; }

            [DataMember(Name = "title")]
            public string Title { get; set; }
        }
    }
}